ginko/src/dts/project.rs

```diff
@@ -335,7 +357,9 @@ mod tests {
         let mut project = Project::default();
         let temp_dir = TempDir::new();
         let (_, path1) = temp_dir.add_file("tests-include.dtsi", "");
-        project.add_file(path1.clone()).expect("Cannot add file");
+        project
+            .add_file(path1.clone().into_os_string().into_string().unwrap())
```

Comment:
Any reason for this change?

Reply:
I've changed the `add_file` signature from `add_file(&mut self, file_name: PathBuf) -> Result<(), io::Error>` to `add_file(&mut self, file_name: String) -> Result<(), io::Error>` since `dunce::canonicalize` already transformed the input `file_name` to a `PathBuf`.